src/styles/cards.scss

```diff
@@ -0,0 +1,22 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
The `.cards` container is using flexbox, but the task requires a grid layout. Please change `display: flex;` to `display: grid;` and adjust the grid properties accordingly.